include/rapidjson/reader.h

```diff
@@ -1039,7 +1040,17 @@ class GenericReader {
             }
             else if (RAPIDJSON_UNLIKELY(static_cast<unsigned>(c) < 0x20)) { // RFC 4627: unescaped = %x20-21 / %x23-5B / %x5D-10FFFF
                 if (c == '\0')
-                    RAPIDJSON_PARSE_ERROR(kParseErrorStringMissQuotationMark, is.Tell());
+					RAPIDJSON_PARSE_ERROR(kParseErrorStringMissQuotationMark, is.Tell());
+				else if (!isKey && c == '\n' && (parseFlags & kParseMultiLineValueFlag)) {
+					is.Take();
+					os.Put('\\');
+					os.Put('n');
+				}
+				else if (!isKey && c == '\t' && (parseFlags & kParseMultiLineValueFlag)) {
```

Comment:
What about other whitespace such as ' ' space character? Maybe Creator used spaces instead of tabs - again: how did previous json loading handle that?